beebop/app.py

```diff
@@ -147,51 +154,74 @@ def add_files(
 
 
 @app.errorhandler(500)
-def internal_server_error(e) -> json:
+def internal_server_error(e) -> tuple[Response, Literal[500]]:
     """
     :param e: [error]
-    :return json: [error response object]
+    :return Response: [error response object]
     """
+    logger.warning(f"Internal Server Error: {e}")
```

Comment:
It looks like you can use the exception method to log exception info, but I'm not actually sure how you provide it!
https://docs.python.org/3/library/logging.html#logging.Logger.exception
```suggestion
    logger.exception(f"Internal Server Error: {e}")
```